### PR TITLE
Simplify test scripts

### PR DESCRIPTION
**Pull request by vivianjeng**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint --check`) has passed locally and any fixes were made for failures

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): Test scripts modification

## What is the current behavior?

Too many duplicated utils functions in both `core`, `contracts` and `circuits`, it is annoying to handle the same functions which are used in three tests.
And it has to be downloaded to perform test scripts so I think it can be used cross different packages

Issue Number: N/A

## What is the new behavior?

- Remove same utils functions in `contracts`, `core`
- Fix test scripts
-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/circuits/test/utils.ts [hunk 1]
```diff
@@ -266,7 +267,10 @@ const genStartTransitionCircuitInput = (
     return crypto.stringifyBigInts(circuitInputs)
 }
 
-const bootstrapRandomUSTree = async (): Promise<crypto.SparseMerkleTree> => {
+const bootstrapRandomUSTree = async (): Promise<{
+    userStateTree
+    reputationRecords
```
Comment (vuvoth) [inline packages/circuits/test/utils.ts:272]: I think we should add types to parameters and void to use `any`.

File: packages/core/test/UnirepState/reputationProofEvent.test.ts [hunk 2]
```diff
@@ -96,29 +89,6 @@ describe('Reputation proof events in Unirep State', function () {
         })
     })
 
-    describe('Init Unirep State', async () => {
```
Comment (vuvoth) [inline packages/core/test/UnirepState/reputationProofEvent.test.ts:99]: I think we should move `Init Unirep State` test suit to `InitUnirep.test.ts`(new one)

File: packages/circuits/test/utils.ts (hunk 1 above)
Comment (vivianjeng) [reply packages/circuits/test/utils.ts:272]: what if we define type here?
```typescript
type UserStates = {
    userStateTree: SparseMerkleTree,
    reputationRecords: { [key: string]: IReputation }
}
```

Comment (vuvoth) [reply packages/circuits/test/utils.ts]: Yes, we should avoid you `any`

**Review by vuvoth — APPROVED** [review]

**Review by vimwitch — APPROVED** [review]

**Pull request merged**